.github/workflows/ci.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.18.x,1.19.x]
+        go-version: [1.19.x, 1.20.x]
```

Comment:
We should bump the version in `go.mod`, too.

Reply:
Should we? This package isn't using any language features that aren't available in 1.18, so it should compile correctly. Bumping `go.mod` _forces_ users to update, which is maybe a little extreme for routine CI maintenance?

Reply:
Then shouldn't we keep `1.18.x` in the set of tested versions, to make sure we don't inadvertently add anything incompatible?

Reply:
My thinking on this has been that we only support (i.e., entertain bug reports for) the last two releases of Go, so we don't need to test any further back. If packages _happen_ to work on older versions, that's a happy coincidence for any slow-to-upgrade users.

In this particular case, though, the latest Connect doesn't support Go 1.18 anyways, so bumping `go.mod` is fine.